### PR TITLE
245 Update tutorials with new API (part 2)

### LINKED ISSUE #245: Update all the examples & notebooks to use new APIs
The feature comes from: https://github.com/Project-MONAI/MONAI/issues/2402

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

Partially fixes #245 .

### Description
This PR modifies the remnant examples in `modules` (as mentioned in #246 ).

### Status
**Ready**

### Checks

- [x] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
Hi @SachidanandAlle , could you please help to update the `DeepgrowStatsHandler ` (https://github.com/Project-MONAI/tutorials/blob/master/deepgrow/ignite/handler.py#L55) to meet the decollate changes? Thanks!

**Comment by SachidanandAlle** [comment]
I suggest we discontinue this Handler.. Developer can use existing ones to write dice and dump images..

**Comment by yiheng-wang-nv** [comment]
> I suggest we discontinue this Handler.. Developer can use existing ones to write dice and dump images..

Do you mean remove the file `handler.py`?

**Comment by SachidanandAlle** [comment]
Yes..

File: modules/jupyter_utils.ipynb [hunk 1]
```diff
@@ -170,9 +172,9 @@
     "\n",
     "\n",
     "def _meandice_transform(x):\n",
-    "    y_pred = x[CommonKeys.PRED]\n",
-    "    y_pred = predict_segmentation(y_pred)\n",
-    "    return y_pred, x[CommonKeys.LABEL]\n",
+    "    y_pred = [predict_segmentation(i[CommonKeys.PRED]) for i in x]\n",
```
Comment (Nic-Ma) [inline modules/jupyter_utils.ipynb:175]: Should use `decollate_batch()` instead of `for i in x`?

File: modules/postprocessing_transforms.ipynb [hunk 2]
```diff
@@ -625,15 +593,15 @@
     "        plt.imshow(val_data.detach().cpu()[0, 0, :, :, 80], cmap=\"gray\")\n",
     "        plt.subplot(1, 5, 2)\n",
     "        plt.title(f\"argmax {i}\")\n",
-    "        argmax = AsDiscrete(argmax=True)(val_output)\n",
+    "        argmax = torch.stack([AsDiscrete(argmax=True)(i) for i in decollate_batch(val_output)])\n",
```
Comment (Nic-Ma) [inline modules/postprocessing_transforms.ipynb:596]: no need to `torch.stack()` I think?

File: modules/postprocessing_transforms.ipynb [hunk 3]
```diff
@@ -625,15 +593,15 @@
...
     "        plt.imshow(argmax.detach().cpu()[0, 0, :, :, 80])\n",
     "        plt.subplot(1, 5, 3)\n",
     "        plt.title(f\"largest {i}\")\n",
-    "        largest = KeepLargestConnectedComponent(applied_labels=[1])(argmax)\n",
+    "        largest = torch.stack([KeepLargestConnectedComponent(applied_labels=[1])(i) for i in decollate_batch(argmax)])\n",
```
Comment (Nic-Ma) [inline modules/postprocessing_transforms.ipynb:600]: no need to `torch.stack()` I think?

File: modules/postprocessing_transforms.ipynb [hunk 4]
```diff
@@ -625,15 +593,15 @@
...
     "        plt.imshow(largest.detach().cpu()[0, 0, :, :, 80])\n",
     "        plt.subplot(1, 5, 4)\n",
     "        plt.title(f\"contour {i}\")\n",
-    "        contour = LabelToContour()(largest)\n",
+    "        contour = torch.stack([LabelToContour()(i) for i in decollate_batch(largest)])\n",
```
Comment (Nic-Ma) [inline modules/postprocessing_transforms.ipynb:604]: no need to `torch.stack()` I think?

**Review by Nic-Ma — COMMENTED** [review]
Thanks for the quick update, I put some comments inline.
Others look good to me.

Thanks.

File: modules/jupyter_utils.ipynb (hunk 1 above)
Comment (yiheng-wang-nv) [reply modules/jupyter_utils.ipynb:175]: Hi Nic, this line is used in metric calculation step, which is after the decollate step in workflow.

**Review by Nic-Ma — APPROVED** [review]
Thanks for the quick update.
Looks good to me now.
@wyli could you please help double confirm it?

Thanks in advance.

**Review by wyli — APPROVED** [review]
Looks good to me
